org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/Extensions.java

```diff
@@ -254,5 +257,65 @@ public static IMappedFormFactory getMappedFormFactory(){
 		}
 		return formFactoryService;
 	}
-
+	
+	private final static CCache<String, IServiceReferenceHolder<IQuickEntryFactory>> s_quickEntryFactoryCache = new CCache<>(null, "IFormFactory", 100, false);
```

Comment:
wrong cache name

Reply:
I tried to deduct the naming convention from the existing caches. If I was wrong what would be the correct name (and the convention)? 
Key type will be changed to Integer.

Reply:
not the variable name but "IFormFactory" which is obviously wrong here.